README.md

```diff
@@ -66,6 +66,26 @@ dev/test
 
 These tests provide a full suite of unit and integration tests for the `xmtpd` repo to help ensure and maintain correctness of the code over time and to avoid regressions as the code evolves. You can explore the tests by taking a look at any files with the suffix `_test.go`. 
 
+## Connect the node to the experimental XMTP testnet
+
+You can connect the `xmtpd` node to an experimental XMTP testnet.
+
+You can use https://pizza.xmtp.chat/ to try messaging on the XMTP testnet.
+
+**Warning:** The XMTP testnet is unstable and we regularly delete data from it.
+
```

Comment:
@snormore - Created this PR to add info about how to connect the node to the testnet. Added some placeholder content below - not sure if it is in the correct format or location. But when you have time, might you be able to help provide the steps/instructions to make the connection? I can try them out and we'll be on our way! Thank you for all of your support and guidance. Such an exciting time...

Reply:
Ah I'm realizing now that there's some minor work to do to support external connections to the pizzanet. I opened an issue to track it here: https://github.com/xmtp-labs/xmtpd-terraform/issues/28. Will follow-up here with the steps/instructions.